geode-core/src/test/java/org/apache/geode/internal/net/BufferPoolTest.java

```diff
@@ -15,78 +15,80 @@
 
 package org.apache.geode.internal.net;
 
+import static org.apache.geode.util.internal.GeodeGlossary.GEMFIRE_PREFIX;
```

Comment:
Since the symbol for GEMFIRE_PREFIX is internal, any chance we can use this opportunity to scrub it from the codebase and use GEODE_PREFIX instead?

Is it worth it?

Reply:
One is "gemfire." and the other is "geode.". So if we changed  this one, for example, to GEODE_PREFIX we would break anyone using the old sys prop name. So although the constant is internal it's value is part of the external contract we have with users.